app.js

```diff
@@ -1,4 +1,7 @@
 const productsCenterDOM = document.querySelector(".products-center");
+let cartDOM = document.querySelector(".cart-btn");
```

Comment:
@SabaMallah I think it should be const not let because you are not updating the variable. Also, the variable name should be consistent with the class name. I suggest cartBtn (no DOM)